code/game/objects/effects/temporary_visuals/clockcult.dm

```diff
@@ -249,9 +249,10 @@
 	light_power = 0.7
 	duration = 55
 
-/obj/effect/temp_visual/ratvar/warp_marker/Initialize(mapload, mob/living/servant)
+/obj/effect/temp_visual/ratvar/warp_marker/Initialize(mapload, mob/living/servant, _duration = 50)
+	duration = _duration + 5
```

Comment:
```suggestion
	duration = _duration + 5
```
**_duration** is an absolutely disgusting variable name, a more descriptive name would be appropriate, at the very least the var name should not start with an _ and should be clearly at a glance different than every other variable name in the relevant part.

Reply:
_duration is used to set the effect's duration (thus the name) which is offset by 5 as the duration was already offset from the animation's time by 5 for a reason I do not currently know but also do not want to conflict. It does not exist outside this one usage of setting the actual duration variable and animation time.